governance/polls/MIP9 Community Greenlight Poll - TUSD-PSM (TrueUSD) - January 10, 2022.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - TUSD-PSM (TrueUSD) - January 10, 2022
+summary: Signal your support or opposition to prioritising onboarding TUSD-PSM (TrueUSD). 
+discussion_link: https://forum.makerdao.com/t/tusd-mip6-collateral-onboarding-application/12192
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Greenlight
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2022-01-10T16:00:00
+end_date: 2022-01-24T16:00:00
+---
+# Poll: Community Greenlight Poll - TUSD-PSM (TrueUSD) - January 10, 2022
+
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP9: Community Greenlight](https://mips.makerdao.com/mips/details/MIP9). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, January 10 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether applying a PSM using TUSD as collateral should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state). 
```

Comment:
Could you change "applying to adding" and then I think it's good to go after that